extensions/binding/org.eclipse.smarthome.binding.ble.yeelightblue/src/main/java/org/eclipse/smarthome/binding/ble/yeelightblue/YeeLightBlueBindingConstants.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.ble.yeelightblue;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link Ble.YeeLightBlueBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Chris Jackson - Initial contribution
+ */
+public class YeeLightBlueBindingConstants {
+
+    private static final String BINDING_ID = "ble";
```

Comment:
You should not re-define it here, but simply use `BleBindingConstants.BINDING_ID` instead.